doc/source/whatsnew/v2.0.0.rst

```diff
@@ -479,6 +481,13 @@ The alternative options giving similar functionality are indicated below:
   ``styler.render.max_rows``, ``styler.render.max_columns`` and
   ``styler.render.max_elements``.
 
+Note that due to this change some defaults have also changed:
+
+- ``multirow`` now defaults to *True*.
+- ``multicol_align`` defaults to *"r"* instead of *"l"*.
+- ``multicol_align`` defaults to *"r"* instead of *"l"*.
```

Comment:
should be multi row

Reply:
Curious why these defaults needed to change.

Reply:
because the styler version defaults are different. since we are merging and have been recommending user to styler anyway it doesnt make sense to change the styler defaults instead of these